src/components/Header/index.jsx

```diff
@@ -23,43 +23,43 @@ import styles from './Header.module.css';
 
 const ForwardNavLink = forwardRef((props, ref) => <NavLink innerRef={ref} {...props} />);
 
-const GordonHeader = ({ onDrawerToggle }) => {
+// Tab url regular expressions must be listed in the same order as the tabs, since the
+// indices of the elements in the array on the next line are mapped to the indices of the tabs
+const TabUrlPatterns = [
+  /^\/$/,
+  /^\/involvements\/?$|^\/activity/,
+  /^\/events\/?$/,
+  /^\/people$|^\/myprofile|^\/profile/,
+  /^\/timesheets$/,
+];
+
+/**
+ * Update the tab highlight indicator based on the url
+ *
+ * The checks use regular expressions to check for matches in the url.
+ */
+const updateTabHighlight = () => {
```

Comment:
```suggestion
const useTabHighlight= () => {
```

Can we change the name of this to clarify that it's a custom hook.

Reply:
I see. Will do!